pkg/api/folder_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	"github.com/xorcare/pointer"
```

Comment:
💡

Reply:
But also an additional dependency with 35 stars on github and without updates for the past few years. Although, the only line of code we actually need from there is `func String(v string) *string { return &v }` (which we could just have as a private help function in this file)

Reply:
the dependency [is not introduced](https://github.com/grafana/grafana/blob/06a2943cb667f140da9ac33368315bf4407911f5/go.mod#L99) in this PR though

Reply:
I have removed it in any case; please check once again